CMakeLists.txt

```diff
@@ -22,6 +22,11 @@ find_program(CLANG_FORMAT_BIN clang-format REQUIRED)
 
 include(GoogleTest)
 
+# Set the default install location to /usr/lib64 on 64-bit systems
+if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+    set(CMAKE_INSTALL_LIBDIR lib64)
```

Comment:
This won't work on multiarch distros like Debian. I think there's a cmake macro you can use to pick up the correct lib path

Reply:
Thanks for pointing it out! It should be fixed in #8 :)